src/views/Game/Game.tsx

```diff
@@ -2612,6 +2677,67 @@ export class GameChat extends React.PureComponent<GameChatProperties, any> {
             </div>
         );
     }}}
+
+    renderQC = (user) => {{{
+        let quick_chat: JSX.Element = null;
+
+        if (this.state.qc_visible) {
+            if (user.qc_phrases === undefined) {
+                let qc_local = localStorage.getItem("ogs.qc.messages");
+                if (qc_local === null) {
+                    let qc_messages: string[] = [
+                        "Hi",
```

Comment:
Can we localize these too? (Should just be a matter of wrapping them like `_("Hi")`

Reply:
Ah, yeah the reason why we need to do it explicitly for every string is because the string extractor looks for _("blah") calls and adds "blah" to the list of strings to be translated, otherwise those strings won't make it to the translators to be translated. If the string is already somewhere in our translation dictionary then what you had would work, but those are new strings so we gotta wrap em :)

Reply:
Okay, gotcha. I'll commit the change now.